instance/templates/instance/ansible/swift.yml

```diff
@@ -0,0 +1,6 @@
+EDXAPP_SWIFT_AUTH_VERSION: '2'
+EDXAPP_SWIFT_USERNAME: '{{ user }}'
+EDXAPP_SWIFT_KEY: '{{ password }}'
+EDXAPP_SWIFT_TENANT_NAME: '{{ tenant }}'
+EDXAPP_SWIFT_AUTH_URL: '{{ auth_url }}'
+EDXAPP_SWIFT_REGION_NAME: '{{ region }}'
```

Comment:
@mtyaka Right, we need to add `EDXAPP_SETTINGS: openstack` to vars.yml, but can't do this right now, since it would break provisioning any branch that doesn't include edx/edx-platform#11286.  I'm not sure how to handle this transition.  For now, I'd suggest to manually add this to the PR settings if we need it, since I can't think of a good way to automatically decide which set of settings to use.

We won't need to set `EDXAPP_DEFAULT_FILE_STORAGE` explicitly, since the is implied by `EDXAPP_SETTINGS: openstack` provided you pass in Swift credentials, but we will need these settings as well:

    EDXAPP_SETTINGS: 'openstack'
    XQUEUE_SETTINGS: 'openstack_settings'
    COMMON_VHOST_ROLE_NAME: 'openstack'